flickit-assessment-data/src/main/resources/liquibase/changelog-0.37.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                                       http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.17.xsd">
+
+    <changeSet id="0.37-01" author="Maziyar Gerami">
+        <addColumn tableName="fak_kit_version">
+            <column name="status_version" type="bigint" valueComputed="
+            CASE
+                WHEN status = 0 THEN 0
+                WHEN status = 1 THEN 1
+                ELSE -id
+            END"/>
```

Comment:
**Suggestion:** Refine the SQL CASE expression in the `addColumn` changeSet to ensure it handles all potential `status` values correctly and efficiently, avoiding hard-coded values. [general, importance: 6]
```suggestion
            <column name="status_version" type="bigint" valueComputed="
            CASE
                WHEN status IS NULL THEN NULL
                WHEN status IN (0, 1) THEN status
                ELSE -id
            END"/>
```